src/Sources/linux.common/mount.cs

```diff
@@ -0,0 +1,75 @@
+using System.Runtime.InteropServices;
+using static Tmds.Linux.LibraryNames;
+
+namespace Tmds.Linux
+{
+    public static unsafe partial class LibC
+    {
+        public static int MS_RDONLY = 1;
+
+        public static int MS_NOSUID = 2;		
+
+        public static int MS_NODEV = 4;			
+
+        public static int MS_NOEXEC = 8;		
+
+        public static int MS_SYNCHRONOUS = 16;		
+
+        public static int MS_REMOUNT = 32;		
+
+        public static int MS_MANDLOCK = 64;		
+
+        public static int MS_DIRSYNC = 128;		
+
+        public static int MS_NOATIME = 1024;		
+
+        public static int MS_NODIRATIME = 2048;		
+
+        public static int MS_BIND = 4096;		
+
+        public static int MS_MOVE = 8192;
+
+        public static int MS_REC = 16384;
+
+        public static int MS_SILENT = 32768;
+
+        public static int MS_POSIXACL = 1 << 16;	
+
+        public static int MS_UNBINDABLE = 1 << 17;	
+
+        public static int MS_PRIVATE = 1 << 18;		
+
+        public static int MS_SLAVE = 1 << 19;		
+
+        public static int MS_SHARED = 1 << 20;		
+
+        public static int MS_RELATIME = 1 << 21;
+
+        public static int MS_KERNMOUNT = 1 << 22;
+
+        public static int MS_I_VERSION =  1 << 23;
+
+        public static int MS_STRICTATIME = 1 << 24;	
+
+        public static int MS_ACTIVE = 1 << 30;
+
+        public static int MS_NOUSER = 1 << 31;
+
+        public static int MNT_FORCE = 1;
+        
+        public static int MNT_DETACH = 1;
+        
+        public static int MNT_EXPIRE = 1;
+        
+        public static int UMOUNT_NOFOLLOW = 1;
```

Comment:
Can you change these into properties, that is `=>` instead of `=`, and also use the `ulong_t` type.

Reply:
If use the `ulong_t` type, `MS_NOUSER` will get an error. Any ideas?

Reply:
The values are wrong.
```
_Static_assert(MS_NOUSER == 2147483648, "MS_NOUSER has unexpected value");
 ^
/tmp/2eac0d2b-6336-4378-8386-4b15817fcfc8.c:1435:1: error: static assertion failed: "MNT_DETACH has unexpected value"
 _Static_assert(MNT_DETACH == 1, "MNT_DETACH has unexpected value");
 ^
/tmp/2eac0d2b-6336-4378-8386-4b15817fcfc8.c:1438:1: error: static assertion failed: "MNT_EXPIRE has unexpected value"
 _Static_assert(MNT_EXPIRE == 1, "MNT_EXPIRE has unexpected value");
 ^
/tmp/2eac0d2b-6336-4378-8386-4b15817fcfc8.c:1441:1: error: static assertion failed: "UMOUNT_NOFOLLOW has unexpected value"
 _Static_assert(UMOUNT_NOFOLLOW == 1, "UMOUNT_NOFOLLOW has unexpected value");
 ^
```
UMOUNT_NOFOLLOW should be 8
MNT_EXPIRE should be 4
MNT_DETACH should be 2

MS_NOUSER is a challenge, glibc defines it as 1 << 31, while it should be 1U << 31.
You can add it to this lists of constants which get skipped:
https://github.com/tmds/Tmds.LibC/blob/6a270975be475b1f4dc55d70242314496408e29d/test/Tmds.LibC.Tests/ConstantsTests.cs#L49
Add a comment after it `// glibc defines it as 1 << 31, while it should be 1U << 31.`.